docker/gcp_pudl_etl.sh

```diff
@@ -89,8 +95,12 @@ run_pudl_etl 2>&1 | tee $LOGFILE
 # Notify slack if the etl succeeded.
 if [[ ${PIPESTATUS[0]} == 0 ]]; then
     notify_slack "success"
+    copy_outputs_to_gcs
```

Comment:
In the old version we copy to GCS regardless of success, so we can see the failed outputs etc. - do you think that behavior still makes sense?

Reply:
Yes, indeed it does as was demonstrated in the recent rounds of failures.